day_7/10_little_monkeys.js

```diff
@@ -0,0 +1,51 @@
+//First Part:
+var numberOfMonkeys = {10: 'Ten', 9: 'Nine', 8: 'Eight', 7: 'Seven', 6: 'Six', 5: 'Five', 4: 'Four', 3: 'Three', 2: 'Two'};
```

Comment:
@cameronRomo - Excellent work here! I like how you mapped the numeric values to their string equivalents in an object. Looks great overall!